reconciler/reconciler.go

```diff
@@ -372,7 +388,7 @@ func (r *Reconciler) CompareBalance(
 		)
 	}
 
-	difference, err := types.SubtractValues(computedBalance.Value, amount)
+	difference, err := types.SubtractValues(amount, computedBalance.Value)
```

Comment:
Did previous consumers of this function not care about which of these was greater?

Reply:
No they do not (the only known caller is `rosetta-cli` which just cares if it failed or not).

Reply:
Great question!